style.css

```diff
@@ -1,4 +1,36 @@
 /* Write your CSS here! */
+.header { 
+  display: grid;
+  grid-template-columns: 130px 400px;
+}
+
+.primary-content {
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  grid-auto-rows: minmax(100px, auto);
+  min-height: 100%;
+}
+
+.footer {
```

Comment:
Try to think about how you can achieve the desired result without using position: fixed?